week1/calculus.ipynb

```diff
@@ -0,0 +1,1060 @@
+{
```

Comment:
<p>Also, mention the application of Differentiation in Neural Networks. For e.g. Differentiation is used to see the rate of change in the loss when a small change is made in weights/bias in the X layer in backward pass.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/toggle-corp/ml-internship/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>